src/components/SurprisePersona.js

```diff
@@ -0,0 +1,130 @@
+    <div className="filterCombo">
+        <div className="circle">
+            <FontAwesomeIcon icon={filter.icon} />
+        </div>
+        <span className="filter-label">{filter.label}</span>
+    </div>
+)
+
+const renderSlide = (slide) => (
+    <a className="link-nostyle" href="/recipe-page">
+        <div className="img" style={{ 'background-image': `url(${slide.imageURL})`}}>
+            <p>  </p>
+        </div>
+        <p className="recipe-name">{slide.recipeName}</p>
+    </a>
+  );
+
+class SurprisePersona extends Component {
+    constructor(props) {
+    super(props);
+    this.state = {
+        filters: [
+            {
+                label: '$10/meal',
+                icon: faDollarSign,
+            },
+            {
+                label: '3hrs/week',
+                icon: faClock,
+            },
+            {
+                label: 'Basic Recipes',
+                icon: faStar,
+            },
+            {
+                label: 'All Ingredients',
+                icon: faUtensils,
+            },
+            {
+                label: 'No Restrictions',
+                icon: faFlag,
+            },
+        ],
+        index: 0,
+    };
+  }
+
+
+  goToPrevious() {
+    if (this.state.index > 0) {
+      this.setState({index: this.state.index-1})
+    }
+  }
+
+  goToNext() {
+    const { recipesMaster } = this.props
+    if (this.state.index < recipesMaster.length-1) {
+      this.setState({index: this.state.index+1})
+    }
+  }
+
+
+  render() {
+    const { recipesMaster } = this.props
+    return(
+        <div>
+            <h1>Recipe for 'Granolas'</h1>
+            <div id="all-filters">
+                <div id="edit">
+                    <FontAwesomeIcon icon={faEdit} />
+                    <a className="link-nostyle" href="/filter">edit</a>
+                </div>
+                <div className="filter">
+                    {R.map(renderFilter, this.state.filters)}
+                </div>
+            </div>
+            
+            {/* carousel */}
+            <div id="carousel-container">
+                <div className="prevBtn" onClick={() => this.goToPrevious()}>
+                    <FontAwesomeIcon icon={faArrowCircleLeft} />
+                </div>
+                <div className="carousel-img">
+                    {renderSlide(recipesMaster[this.state.index])} 
+                </div>
+                <div className="nextBtn" onClick={() => this.goToNext()}>
+                    <FontAwesomeIcon icon={faArrowCircleRight} />
+                </div>
+            </div>
+
+            <button>
+                <a href="/recipes-all">View All Recipes</a>
+            </button>
+        </div>
+    )
+  }
+}
+
+
+function mapStatetoProps(state) {
+  return {
+    recipesMaster: state.recipesMaster,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators({}, dispatch);
+}
+
+export default connect(mapStatetoProps, mapDispatchToProps)(SurprisePersona);
```

Comment:
Is the surprise persona page just the same as the customize surprise page?

Reply:
The surprise page can just be the same one — in the persona page we'd just send along some presets so that the filters are already set